libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -203,18 +206,30 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
         this._useForm = coerceBooleanProperty(value);
     }
 
+    /**
```

Comment:
Remove above `multilayout` getters unless you see the use for it .If you see there are some leftover and there is something you dont watch to touch let us know. we need to have under control every line